pkg/model/contextresource.go

```diff
@@ -50,6 +51,10 @@ func GetContextResource(path string) (*ContextResource, error) {
 		return nil, newManifestFriendlyError(err)
 	}
 
+	if ctxResource.Context != "" || ctxResource.Namespace != "" {
```

Comment:
NIT: It's weird to me that I set both of the fields written even when I just wrote one of them in the manifest.

Reply:
@jLopezbarb I initially preferred to keep it simple, but you are right, it can be confusing if someone receives a warning of something that they are NOT doing. I've changed the code in 59a654ea963fe0ab36e0d251cda48bb772ff120f

Please have another look 🙏